server/routes/pollRouter.js

```diff
@@ -0,0 +1,45 @@
+const router = require('express').Router();
+const { cloudinary } = require('../utils/cloudinary');
+const Poll = require('../models/pollModel');
+
+router.post('/upload', async (req, res) => {
+    try {
+        let { imageOne, imageTwo, question, friendlist, userId } = req.body;
```

Comment:
```suggestion
        const { imageOne, imageTwo, question, friendlist, userId } = req.body;
```

This can be a const cause it won't change